mod/account/spec/set/right/password_spec.rb

```diff
@@ -17,14 +17,13 @@ def card_subject
 
   describe "#update" do
     it "encrypts password", aggregate_failures: true do
-      password_card.update! content: "new password"
+      password_card.update! content: "new Pas5word!"
       expect(password).not_to eq("new password")
-      authenticated = Card::Auth.authenticate "joe@user.com", "new password"
+      authenticated = Card::Auth.authenticate "joe@user.com", "new Pas5word!"
       expect(account).to eq authenticated
     end
 
     it "validates password" do
-      password_card = account.password_card
       password_card.update content: "2b"
       expect(password_card.errors[:password]).not_to be_empty
```

Comment:
I would like a little more sophisticated testing here. Can you show that the length and character requirements are both working? I would rather we not just test for the presence of errors but also that the errors are correct.